includes/validation/class-amp-validation-manager.php

```diff
@@ -600,6 +600,14 @@ public static function init_validate_request() {
 		if ( true === $should_validate_response ) {
 			self::add_validation_error_sourcing();
 			self::$is_validate_request = true;
+
+			$display_errors = ini_get( 'display_errors' );
+
+			if ( ! empty( $display_errors ) || 'stderr' !== $display_errors ) {
```

Comment:
If the value of `display_errors` is set to `stderr` it will not be send to `stdout`, as documented in the [manual](https://www.php.net/manual/en/errorfunc.configuration.php#ini.display-errors).